### PR TITLE
Group deployment of an application from Esper Plugin

### PR DESCRIPTION
**Pull request by AshokKumarMandava**
Added the documentation for Group deployment of an application from Esper Plugin.
Updated the images for Setup section and Upload APK success case.

### CODE REVIEW

File: docs/home/esperplugin.md [hunk 1]
```diff
@@ -67,6 +70,40 @@ If there's any error, you'll see an error as shown below:
...
+## Deploying application to a group of devices
+Esper plugin faciliates developers to deploy an uploaded applciation to a group.
+
+### After uploading APK
+After [uploading the APK to Esper](#uploading-apk-to-esper), developer can opt to deploy to a group of devices.
+
+<img src="./assets/plugin/deployapptogroup/deploy_after_upload.jpg">
+
+### Uploaded applications
+Developers can choose an application and a group.\
```
Comment (bmehendale) [inline docs/home/esperplugin.md]: (nit) lot of `\`'s

Comment (kirananto) [reply docs/home/esperplugin.md]: It's required for pagebreak..
image

File: docs/home/esperplugin.md [hunk 2]
```diff
@@ -67,6 +70,40 @@ If there's any error, you'll see an error as shown below:
 Note that trying to upload the same version of the .apk you already have on your Endpoint will produce an error. You currently cannot overwrite it using the Plugin, you will have to delete it using the Esper Console or from the CLI.
 
 <br/>
+
+## Deploying application to a group of devices
+Esper plugin faciliates developers to deploy an uploaded applciation to a group.
```
Comment (shivani61) [inline docs/home/esperplugin.md]: *application

File: docs/home/esperplugin.md [hunk 3]
```diff
@@ -37,14 +37,17 @@ If this is the first time you are installing a 3rd-party plugin, Android Studio
 Click on Accept to proceed. If you have installed third-party plugins before, you will not see this prompt, and you can proceed to the next step.
 Once Android Studio restarts, check the Event Log in the lower right corner to verify the Plugin successfully installed.
 
-## Setup and Use
+## Setup
 The setup for the Esper Plugin is Project specific for the currently open Project. If you wish to use the Esper Plugin for other Projects, be sure to perform the setup for each. 
 With your desired Project open, go to File → Settings (or Android Studio → Preferences for Mac). Click on the drop down for Other Settings, and the click on Esper from the resulting list. In the Esper window, type in the relevant credentials for your Endpoint Name, Username, and Password. Hit Apply and then OK.
```
Comment (shivani61) [inline docs/home/esperplugin.md]: in future, we should add a reference to this in our welcome emails to developers @bmehendale @kirananto

File: docs/home/esperplugin.md (hunk 2 above)
Comment (AshokKumarMandava) [reply docs/home/esperplugin.md]: Fixed

File: docs/home/esperplugin.md (hunk 3 above)
Comment (bmehendale) [reply docs/home/esperplugin.md]: @shivani61 - please can you open an internal ticket on @kirananto for this?

**Pull request merged**